src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py

```diff
@@ -71,7 +71,12 @@ def __init__(self, config):
             self.maxRetries = self.config.ErrorHandler.maxRetries
         else:
             self.reqAuxDB = ReqMgrAux(self.config.General.ReqMgr2ServiceURL)
-            self.maxRetries = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName).get("MaxRetries")
+            agentConfig = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName)
+            if (agentConfig.get("UserDrainMode") and agentConfig.get("SpeedDrainMode") and
+                agentConfig.get("SpeedDrainConfig")['NoJobRetries']['Enabled']):
```

Comment:
Seangchan, can you also log that the component is using the MaxRetries=0 here too?